CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 - changed the Scion in The Great Pyramid from spawning blood when hit to a richochet effect if texture fixes enabled (#1121)
 - added the ability to back out of menus with the circle and triangle buttons when using a gamepad (cross acts as confirm) (#1104)
 - changed the gamepad control menu's 'reset all buttons' bind to held R1 (was held triangle) (#1104)
+- expanded the number of visible enemies from 8 to 32 (#1122)
```

Comment:
Let's use the common prefix `changed` and reorder the logs so that `changed the Scion` is with the rest of the `changed` records